event/oracle/approve_oracle_registration_event.go

```diff
@@ -8,18 +8,17 @@ import (
 var _ event.Event = (*ApproveOracleRegistrationEvent)(nil)
 
 type ApproveOracleRegistrationEvent struct {
-	reactor event.Reactor
 }
 
-func NewApproveOracleRegistrationEvent(s event.Reactor) ApproveOracleRegistrationEvent {
-	return ApproveOracleRegistrationEvent{s}
+func NewApproveOracleRegistrationEvent() ApproveOracleRegistrationEvent {
```

Comment:
I have a question. Why remove the reactor for now?

Reply:
For this PR, I don't want to create a service for reactor, which has many things in it.
This `ApproveOracleRegistrationEvent` don't need all that resources in service.
I've been thinking about how to make this `ApproveOracleRegistrationEvent` better. If you have any idea, please feel free to leave your thoughts.

Reply:
I understand it! I'll also thinking about how to make it better. Thanks!